domain-specific-datasets/README.md

```diff
@@ -0,0 +1,17 @@
+# Domain Specific Dataset Project
+
+This is project to bootstrap the creation of domain-specific datasets for training models. The goal is to create a set of tools that help users to collaborate with domain experts.
+
+## Example Farming Project
+
+Here are examples of the resources for our farming example:
+The seeding space: https://huggingface.co/spaces/argilla/domain-specific-seed for the domain expert.
+The demo dataset: https://huggingface.co/datasets/argilla/farming we've created
+The argilla space: https://huggingface.co/spaces/argilla/farming
+The pipeline code: https://github.com/argilla-io/distilabel-workbench/tree/main/projects/farming
```

Comment:
I think the distilabel pipeline (using random LLMs, when to use GPU, when not, where to plugin API keys etc.) is going to be difficult for most users, when not explained well.